README.md

```diff
@@ -20,7 +23,16 @@ For any given task date between two available GPW input images, the estimated po
 ### Class constants
 
     scale = 300
-    POPULATION_DENSITY_THRESHOLD = 1
+    SETTLEMENT_DISTANCE_FROM_COAST = 4000  # meters
+    COASTAL_SETTLEMENT_POPULATION_DENSITY = 10
+    COASTAL_NAVIGATION_DISTANCE = 80000  # meters
+    OCEAN_BUFFER_DISTANCE = 300  # meters
+    GSW_OCCURRENCE_THRESHOLD = 40
+    WIDE_RIVER_MIN_WIDTH = 30  # meters
+    GSW_CONNECTED_PIXEL_MIN = 1024  # pixels
+    INDIRECT_DISTANCE = 15000  # meters
+    DECAY_CONSTANT = -0.0003
+    INDIRECT_INFLUENCE = 10
 
 ## Usage
 
```

Comment:
Need to add `overwrite` arg and any others present in base task (have a quick look at one of the other driver's README)